build.gradle

```diff
@@ -18,6 +17,8 @@ version = "${mod_version}"
 group= "net.tropicraft" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
 archivesBaseName = "tropicraft"
 
+sourceCompatibility = targetCompatibility = '1.8'
```

Comment:
didn't this get added already?

Reply:
It was, here https://github.com/Tropicraft/Tropicraft/pull/142, but that PR was merged into this branch :P